src/VisualStudio/CSharp/Test/Interactive/InteractiveWindowTestHost.cs

```diff
@@ -20,22 +18,12 @@ public sealed class InteractiveWindowTestHost : IDisposable
 
         private readonly System.ComponentModel.Composition.Hosting.ExportProvider _exportProvider;
 
-        internal static readonly IExportProviderFactory ExportProviderFactory = ExportProviderCache.GetOrCreateExportProviderFactory(
-            ExportProviderCache.GetOrCreateAssemblyCatalog(
-                new[]
-                {
-                    typeof(TestWaitIndicator).Assembly,
-                    typeof(TestInteractiveEvaluator).Assembly,
-                    typeof(IInteractiveWindow).Assembly
-                })
-                .WithParts(TestExportProvider.GetCSharpAndVisualBasicAssemblyCatalog())
-                .WithParts(MinimalTestExportProvider.GetEditorAssemblyCatalog()));
+        internal static readonly IExportProviderFactory ExportProviderFactory = EditorTestCompositions.EditorFeaturesInteractiveWindow.ExportProviderFactory;
```

Comment:
💡 Since this is just a pass-through:

```suggestion
        internal static IExportProviderFactory ExportProviderFactory => EditorTestCompositions.EditorFeaturesInteractiveWindow.ExportProviderFactory;
```